scripts/deploy.js

```diff
@@ -358,6 +360,107 @@ async function ensureVerifierDeployment(plan, config) {
     }
 }
 
+// semaphore
+async function ensureSemaphoreVerifierDeployment(plan, config) {
+    await ensureMtbBinary(plan, config);
+    fs.mkdirSync(MTB_CONTRACTS_DIR, { recursive: true });
+    await compileSemaphoreVerifierContract(plan, config);
+    await deploySemaphoreVerifierContract(plan, config);
```

Comment:
All of this should be unnecessary unless there's a reason _not_ to construct the verifier directly in the `initialize` function rather than pass it in from outside.

Reply:
We talked it through via Slack: We decided to take a param with the address to minimize the main contract size.